### ISSUE TITLE
AssertAreEqual analyzer does not trigger if the expected value is null

### ISSUE THREAD

**Issue by robertMSFTE** [fluentassertions/fluentassertions.analyzers#171]
### Description

```AssertAreEqual``` analyzer does not seem to trigger if the expected value is null.

### Complete minimal example reproducing the issue

```csharp
namespace TestProject
{
    using Microsoft.VisualStudio.TestTools.UnitTesting;

    [TestClass]
    public class UnitTest
    {
        [TestMethod]
        public void TestMethod1()
        {
            int? x = null;

            Assert.AreEqual(5, x); // WORKS 
            Assert.AreEqual(null, x); // DOES NOT WORK
        }
    }
}
```

### Expected behavior:

Analyzer triggers and suggests to use fluent assertions.

### Actual behavior:

Analyzer does not show a suggestion.

### Versions

* Which version of Fluent Assertions Analyzers are you using? 0.17.2
* Which .NET runtime and version are you targeting? .NET 6.0.

**Comment by robertMSFTE** [comment]
@Meir017 The scenario still reproduces on ``0.20.0``.

**Comment by Meir017** [comment]
looks like I handled the other case 
```cs
Assert.AreEqual(x, null); // FIXED
Assert.AreEqual(null, x); // DOES NOT WORK
```
this will be fixed shortly
